docs/source/conf.py

```diff
@@ -52,7 +52,7 @@
     + "TransferableFutureCovariatesLocalForecastingModel,GlobalForecastingModel,TorchForecastingModel,"
     + "PastCovariatesTorchModel,FutureCovariatesTorchModel,DualCovariatesTorchModel,MixedCovariatesTorchModel,"
     + "SplitCovariatesTorchModel,TorchParametricProbabilisticForecastingModel,"
-    + "min_train_series_length,uses_future_covariates,uses_past_covariates,"
```

Comment:
Actually, could this become a documented property, which users could query to determine whether a given instance accepts or expects certain covariates? WDYT? Might be useful.